vendor/bat-native-ads/src/bat/ads/ad_content_info.cc

```diff
@@ -164,6 +165,7 @@ bool AdContentInfo::FromValue(const base::Value& value) {
 std::string AdContentInfo::ToJson() const {
   std::string json;
   SaveToJson(*this, &json);
+  BLOG(0, "FOOBAR: " << json);
```

Comment:
FOOBAR? Did we mean to put anything here?

Reply:
Fixed :-)